backend/static/css/style.css

```diff
@@ -279,15 +279,20 @@ li.CodeMirror-hint-active {
   background-color: #FEFEFE;
   border: 1px solid #000;
   min-width: 25em;
-  max-width: 40em;
+  /* max-width: 40em; */
```

Comment:
These can be removed now (was still playing with the look, but it seems OK for now).